README.md

```diff
@@ -53,40 +53,41 @@ sudo node index.js -u all                                 # unset everything exc
 sudo node index.js
 ```
 
+Note that when using glob patterns with `index.js`, you must enclose the pattern in double quotes.
+
 ### Configuration File
 
-Normally a config file is automatically generated (e.g. when you set a key using `-k`), but you can create your own config file `/etc/logdna.conf` on Linux and `C:\ProgramData\logdna\logdna.conf` on Windows:
+Normally a config file is automatically generated (e.g. when you set a key using `index.js -k`) and updated (e.g. when you add a directory using `index.js -d`) but you can create your own config file `/etc/logdna.conf` on Linux and `C:\ProgramData\logdna\logdna.conf` on Windows and save your settings there:
```

Comment:
Creating config files at the specified paths is not necessary because we have `-c` option for specifying the path of the config file - by default, it's `/etc/logdna.conf` (for Unix) or `C:\ProgramData\logdna\logdna.conf` (for Windows).

Reply:
I think the intent was more to indicate that they can edit the conf file at all, and since that's where it resides by default, that's what's listed. How would you phrase that intention differently?

Reply:
@dchai76 

something similar to:
"""
but you can create your own config file and use it by using `index.js -c < Path to Config File >`
"""

Reply:
But, @dchai76, I think mine is kinda nitpicking; therefore, I'm considering this fine!